etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/orders/SendOrderUseCase.java

```diff
@@ -62,17 +65,19 @@ private void saveSentOrderSubmissionId(String receivedSubmissionId, String sentS
                     "Received and/or sent submissionId is null so not saving metadata for sent order");
             return;
         }
-
-        try {
-            partnerMetadataOrchestrator.updateMetadataForSentOrder(
-                    receivedSubmissionId, sentSubmissionId);
-        } catch (PartnerMetadataException e) {
+        Callable<Void> task =
+                () -> {
+                    partnerMetadataOrchestrator.updateMetadataForSentOrder(
+                            receivedSubmissionId, sentSubmissionId);
+                    return null;
+                };
+        boolean taskExecutedSuccessfully = retryTask.retry(task, 3, 1000);
```

Comment:
Can we increment the retry based on counts of retries? Perhaps double or triple the sleep time after each failure?

Reply:
The waitTime already multiplies by 2 each iteration